pkg/chaosdaemon/tc_server.go

```diff
@@ -76,7 +76,7 @@ func (s *daemonServer) SetTcs(ctx context.Context, in *pb.TcsRequest) (*empty.Em
 	}
 
 	client := buildTcClient(ctx, pid)
-	err = client.flush()
+	err = client.flush(in.Device)
```

Comment:
how about setting the `in.Device` to `DefaultDevice` while it is empty. User  may use an older controller manager and without setting the `Device`

Reply:
Fix